js/sudoku.js

```
@@ -0,0 +1,243 @@
+							row2 = Math.floor(Math.random()*3);	
+						}
+						row1 = row1 + no;
+						row2 = row2 + no;			
+						var tmpMatrix = new Array();
+						tmpMatrix = matrix[row1];
+						matrix[row1] = matrix[row2];
+						matrix[row2] = tmpMatrix; 				
+					}			
+				}
+				// Switching columns
+				for(var no=0;no<9;no+=3){
```

Biome (2.1.2)

[error] Shouldn't redeclare 'no'. Consider to delete it or rename it.

'no' is defined here:


(lint/suspicious/noRedeclare)